socket/socket.js

```diff
@@ -62,6 +62,7 @@ const socketEvents = (io) => {
             error: false,
             chatMsg: messageResult.chatMsg,
           });
+          cb();
```

Comment:
what does this callback do?

Reply:
This is for the frontend. If the message successfully sends, it will be added to the conversation view. If not, an error snackbar is shown and the new message won't show up in the conversation.